ci/gpu/build.sh

```diff
@@ -118,6 +118,13 @@ else
         export NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=1
     fi
 
+    # Install contextvars on Python 3.6
+    py_ver=$(python -c "import sys; print('.'.join(sys.version_info[:2]))")
+    if [ "$py_ver" == "3.6" ];then
+        export NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=1
+	conda install contextvars
```

Comment:
```suggestion
        conda install contextvars
```

Reply:
Thanks! Yeah I think the editor added a tab. Fixed :)